src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
```

Comment:
Is noise currently normalised? If not then the default threshold in the PR should not be 0.4?